cmd/tidb-server/main.go

```diff
@@ -1003,3 +1005,25 @@ func closeStmtSummary() {
 		stmtsummaryv2.Close()
 	}
 }
+
+func enablePyroscope() {
+	if os.Getenv("PYROSCOPE_SERVER_ADDRESS") != "" {
```

Comment:
Make it a start parameter or config so that we can integrate it with our infra in the future?

Reply:
Yes， I think it needs to add this config. but this requires the PM's approval.